server/views/dashboard/index.njk

```diff
@@ -5,7 +5,7 @@
 
 {% block content %}
 
-  <h1>This site is under construction...</h1>
-  <p>Please check back later when there is content to view.</p>
+  <h1>{{ pageHeading }}</h1>
 
+  <a class="govuk-link" href="#">Find an accredited programme</a>
```

Comment:
Is there a reason for paremeterising the heading but not the link text?

Reply:
I think the question is more "Is there a reason for parameterising page headings when they're static?"?

Reply:
I think the main reason we parameterise on this project is so we can reuse the `pageHeading` variable for the `pageTitle` above on line 3 (which should be `applicationName + " - " + pageHeading` rather than `applicationName + "- Home`"). We're not currently doing this, but we should be!

Otherwise, there's not a whole lot of advantage to it, other than consistency across the application if we were to have any titles that'd include dynamic content - one use I can imagine would be if we were translating the site we could possibly pass in some translations there, but we might also nunjucks view helpers for that.

Reply:
> We're not currently doing this, but we should be!

Should we? I assumed we keep them separate because we don't want the page title to be as long as the main heading might usefully be?

Yeah, it would help for translations, but then we should do the same for all the text content 😅

I suppose it does keep things consistent regardless of whether it's a static or dynamic/interpolated heading, although arguably it just shifts that distinction to the controller

Reply:
I don't think we need to worry about headings getting all that long to be honest - if they do we can handle it on a case-by-case basis. It looks like [this is the approach](https://github.com/search?q=repo%3Aministryofjustice%2Fhmpps-approved-premises-ui%20pageHeading&type=code) they've taken in the Approved Premises project too